introduction.md

```diff
@@ -1,3 +1,8 @@
-Yes, I am the gitrekt501.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?